### PR TITLE
update DynamoDB definition

### PR DESCRIPTION
**Pull request by hitohata**
## Overview of Implementation

Added a new GSI and an access pattern.

### CODE REVIEW

File: project-document/docs/database/database.md [hunk 1]
```diff
@@ -122,6 +137,7 @@ Food is the object.
 | Access pattern name          | Key (PK/SK)             | How to Access      | Detail                                   | Context   |
 |:-----------------------------|:------------------------|:-------------------|:-----------------------------------------|:----------|
 | GetUser                      | UserId / User           | Get                | Get a single user data                   | User      |
+| GetUserByEMail               | EMailAddress            | Get                | Get a single user data                   | User      |
```
Comment (taniguchiiqqq) [inline project-document/docs/database/database.md:140]: I think this `How to access` should be `Query against GSI`. This GSI means `EMailUserRelation`.

**Review by taniguchiiqqq — COMMENTED** [review]
Thank you. I'm so sorry for such a late review.

File: project-document/docs/database/database.md (hunk 1 above)
Comment (hitohata) [reply project-document/docs/database/database.md:140]: Thank you for your pointing out.
That's true.

**Review by taniguchiiqqq — APPROVED** [review]
Thank you !

**Pull request merged**